rpm/lld.spec

```diff
@@ -0,0 +1,109 @@
+%global maj_ver 14
+%global min_ver 0
+%global patch_ver 6
+
+# Opt out of https://fedoraproject.org/wiki/Changes/fno-omit-frame-pointer
+# https://bugzilla.redhat.com/show_bug.cgi?id=2158587
+%undefine _include_frame_pointers
+
+%global pkg_name lld
+%global install_prefix /usr
+
+Name:		lld
+Version:	%{maj_ver}.%{min_ver}.%{patch_ver}
+Release:	1
+Summary:	The LLVM Linker
+License:	Apache-2.0 WITH LLVM-exception OR NCSA
+URL:		http://llvm.org
+Source:		%{version}/%{name}-%{version}.tar.gz
+
+Patch0:		0006-PATCH-lld-Import-compact_unwind_encoding.h-from-libu.patch
```

Comment:
All patches need to be listed in all spec files for automatic checks to approve the package. Please add the patches 1-5 here and this patch 6 to llvm and clang spec files.